AutomatedTesting/Gem/PythonTests/Performance/TestSuite_Main_DX12.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+"""
+
+# This suite consists of all test cases that are passing and have been verified.
+
+import pytest
+import os
+import sys
+
+from ly_test_tools import LAUNCHERS
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/../automatedtesting_shared')
```

Comment:
Where possible avoid hacking the classpath like this, as it can lead to unexpected collisions. Does a normal relative import fail?

Reply:
I know, and I haven't been able to kill it from growing...